client/utils/index.js

```diff
@@ -244,3 +244,18 @@ export const getErrorMessage = (error, defaultMessage) => {
 
     return defaultMessage
 }
+
+/**
+ * Helper function to determine if the starting and ending dates
+ * occupy entire day(s)
+ * @param {moment} startingDate - A moment instance for the starting date/time
+ * @param {moment} endingDate - A moment instance for the starting date/time
+ * @return {boolean} If the date/times occupy entire day(s)
+ */
+export const isEventAllDay = (startingDate, endingDate) => {
```

Comment:
there is already a function `isAllDay` in this file. It would clearer to merge them